packages/terra-docs/src/Placeholder.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import classNamesBind from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+import styles from './Placeholder.module.scss';
+
+const cx = classNamesBind.bind(styles);
+
+const propTypes = {
```

Comment:
I removed the 'light'/'dark' variants from the placeholder. I believe it's no longer needed since i added theming?